test/smoke_test/smoke_test.py

```diff
@@ -9,4 +9,22 @@
 import torchaudio.sox_effects  # noqa: F401
 import torchaudio.transforms  # noqa: F401
 import torchaudio.utils  # noqa: F401
-from torchaudio.io import StreamReader  # noqa: F401
+
+
+def streamreader_test():
+    from torchaudio.io import StreamReader  # noqa: F401
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--streamreader", action="store_true")
+    parser.add_argument("--no-streamreader", dest="streamreader", action="store_false")
```

Comment:
I think this can be achieved w/ just one argument

Reply:
Done